-meeting-template.md

```diff
@@ -1,57 +1,54 @@
 # YYYY-MM-DD Hypha Worker Co-op: All Hands Meeting
 
-<sup>[from template][template]</sup>
+<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
 
-[:date: Calendar][calendar] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive] | [:notebook: Meeting Index][meetings]
-
-Time: ...  
-Location: https://meet.jit.si/hyphacoop  
-Attending: ...
-
-Notetaker: ...
-
-:raising_hand: Next: ... COPY FROM PREVIOUS WEEK  
-<sup>Bump to end on taking turn :)</sup>
-
-[TOC]
+Time:      ...  
+Location:  https://meet.jit.si/hyphacoop  
+Attending: ...  
+Notetaker: :raising_hand: ei, yj, gi, pc, dc, uv, bl  
 
 ## Agenda
 
-- Previous Action Items
-- Runthrough Announcements and Updates (add below)
-- Quick Working Group Updates (10 mins)
-    - [Business Planning][biz-wg]
-    - [Finance][fin-wg]
-    - [Governance][gov-wg]
-    - [Infra & Ops][ops-wg]
-- Discussion
-    ...
-
-## Announcements
-
-- ...
+| Item                                            | Time (min) |
+|:------------------------------------------------|-----------:|
+| [Previous Action Items][tasks]                  |          5 |
+| [Announcements](#Announcements)                 |          5 |
+| [Working Group Updates](#Working-Group-Updates) |         10 |
+| [Project Updates](#Project-Updates)             |         10 |
+| [Discussions](#Discussions)                     |         30 |
 
 ## Notes
 
+### Announcements
+
 - ...
 
-## Follow Up
+### Working Group Updates
+
+- [Business Planning][biz-wg]
+- [Finance][fin-wg]
+- [Governance][gov-wg]
+- [Infra & Ops][ops-wg]
 
-### Actionable Items
+### Project Updates
```

Comment:
Where do actionable items go?

Reply:
Project board task tracking

Reply:
would be nice if there were a space, as sometimes project board might not be convenient. cultivator can deal with straggling todos in notes before closing out, but having one space for that adds clarity

Would suggest template have:

```
### Action Items
- [ ] cultivator: migrate action items to task board
- [ ] ...
```

Reply:
also, proposing --

1. person speaking:

> `patcon: this is a thing patcon said`

2. assignment of todo:

> - [ ] this is a thing for patcon to do (patcon)

We use (1) for everything now, and it's been ambiguous whether something is assigned or suggested by someone. That distinction could be made clear in template by demonstration, if seems sensible